assets/dev/scss/frontend/widgets/image-carousel.scss

```diff
@@ -21,8 +21,7 @@
 }
 
 // Before the image-carousel layout is calculated by JS. //
-.elementor-image-carousel-wrapper:not(.swiper-container-initialized),
-.elementor-image-carousel-wrapper:not(.swiper-initialized) {
+.elementor-image-carousel-wrapper:not(.swiper-container-initialized):not(.swiper-initialized) {
```

Comment:
Unrelated fix.

Reply:
What this fix relates to?

Reply:
The separate not selectors don't work, because the selectors all always try, which shouldn't be like that. I noticed while working on the Nested Carousel task.